pyproject.toml

```diff
@@ -7,7 +7,7 @@ allow-direct-references = true
 
 [project]
 name = "multiversx-sdk"
-version = "0.13.2"
+version = "0.14.0"
```

Comment:
If title is optional, can increment the last segment (since it's < v1).

Reply:
I will leave it like this as it is not a bug fix